colorama/initialise.py

```diff
@@ -17,7 +18,6 @@
 def reset_all():
     AnsiToWin32(orig_stdout).reset_all()
 
-
```

Comment:
There are two blank lines between functions in this file, so this line shouldn't be removed